eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -3,7 +3,7 @@ jobs:
     variables:
       - template: ../variables/globals.yml
     pool:
-      name: azsdk-pool-mms2019-ds2v2-general # Comment this line back-out to switch to public pools.
+      name: azsdk-pool-mms-win-2019-general # Comment this line back-out to switch to public pools.
```

Comment:
Do we need "mms" and "general" in the pool name? Just trying to see if we can keep the naming convention as simple as possible. With that note does it have to have the "azsdk-pool" prefix as well? I guess the big question is what all do we think we need to encode in the pool name?

Reply:
I suspect all we need is to create a label that represents the image we are using.

Reply:
The reason I have MMS there is that it identies that its the MMS image, as opposed to the standard Azure image for Windows 2019 (which we may end up using in the future). I did this to remove any possible ambiguity.